gkr_iop/src/precompiles/lookup_keccakf.rs

```diff
@@ -137,6 +140,8 @@ pub struct KeccakLayer<WitT, FixedT, EqT> {
     pub wits: KeccakWitCols<WitT>,
     pub fixed: KeccakFixedCols<FixedT>,
     pub eq_zero: EqT,
+    pub eq_mem_read: EqT,
+    pub eq_mem_write: EqT,
```

Comment:
ultimately, we need to tune eq_zero, eq_mem_read, eq_mem_write into right version

- eq_zero => deal with non-pow2 instance
- eq_mem_read => as a selector, only enable on first rotation index per instance 
- eq_mem_write => as a selector, only enable on last rotation index per instance